lib/pg_eventstore/extensions/options_extension.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'set'
+
+module PgEventstore
+  module Extensions
```

Comment:
Nice, but why does it live in PgEventstore? Has nothing to do with it. I would extract it.

Reply:
> Nice, but why does it live in PgEventstore? Has nothing to do with it.

It is a helper module that helps in the implementation of other classes, like `PgEventstore::Event`, `PgEventstore::Config`.

> I would extract it.

It is too tiny to extract.